integrations/operator/CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+## Contributing
+
+### Adding support for new Teleport resources
+
+The steps to add support for new Teleport types to the operator are as follows:
+
+#### Make sure the existing CRDs are up to date
+
+In a clean repo before making any changes:
+
+1. Run `make manifests`.
+2. Run `make -C crdgen update-protos`.
+3. Run `make -C crdgen update-snapshot`.
+
+Make sure everything looks sane and commit any changes (there will be some if
+other .proto files used to generate the CRDs have changed).
```

Comment:
Don't we prevent that in CI?
If not, can we create an issue to track that?

Reply:
CRD geenration pipeline is far from ideal but it's currently so messy that I think it's saner to avoid synchronizing the `crdgen` test data with the upstream proto. real CRDs are generated from upstream protos anyway, this is only for development.

I'd like to get rid of those proto files and snapshots completely.